styles.css

```diff
@@ -139,4 +139,4 @@ span > img {
 
 .social-links > ul > li > a > img {
   margin-right: 15px;
```

Comment:
- [ ] Kindly add the descriptive tittle for your project.

- [ ] Kindly align the logo and the  hamburger button on the same line.
![port15](https://user-images.githubusercontent.com/55635977/129701465-d67636c2-ea9c-44dc-8939-2da422bc7b49.png)

- [ ] Change the `p` tag with `h1` tag . 
![port16](https://user-images.githubusercontent.com/55635977/129701693-c95ebe57-07bb-4e5c-afd7-0af7e67d2b5f.png)